.env.development.local

```diff
@@ -0,0 +1,2 @@
+MONGODB_URI=mongodb+srv://readUser:PEIiFE7YuQ5vGr6u@cluster0.cq1ho.mongodb.net/data?retryWrites=true&w=majority
```

Comment:
see comment in .gitignore this type of file should not appear in the public repo.

You can put this in the google drive.